src/frontend/src/apis/onboarding.api.ts

```diff
@@ -0,0 +1,21 @@
+import { Checklist } from 'shared';
+import { apiUrls } from '../utils/urls';
+import axios from '../utils/axios';
+
+export const getAllChecklists = () => {
```

Comment:
weve typically had comments for api calls and hooks, how useful they are is up for debate